akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/Routers.scala

```diff
@@ -21,8 +21,8 @@ object Routers {
    * before the group detects this. Because of this it is best to unregister routees from the receptionist and not stop
    * until the deregistration is complete to minimize the risk of lost messages.
    */
-  def group[T](key: ServiceKey[T]): GroupRouter[T] =
-    new GroupRouterBuilder[T](key)
+  def group[T](key: ServiceKey[T], localAffinity: Boolean = false): GroupRouter[T] =
```

Comment:
It shouldn't be an optional parameter here but instead default to false and use the `GroupRouter` API to allow for changing it. Default parameters must be avoided in public APIs as they are close to impossible to evolve in a binary compatible way.